packages/@tailwindcss-node/src/compile.ts

```diff
@@ -62,3 +84,49 @@ async function importModule(path: string): Promise<any> {
     throw error
   }
 }
+
+const cssResolver = EnhancedResolve.ResolverFactory.createResolver({
+  fileSystem: new EnhancedResolve.CachedInputFileSystem(fs, 4000),
+  useSyncFileSystemCalls: true,
+  extensions: ['.css'],
+  mainFields: ['style'],
+  conditionNames: ['style'],
+})
+async function resolveCssId(id: string, base: string): Promise<string | false | undefined> {
+  if (typeof globalThis.__tw_resolve === 'function') {
+    let resolved = globalThis.__tw_resolve(id, base)
+    if (resolved) {
+      return Promise.resolve(resolved)
+    }
+  }
+
+  try {
+    let dotResolved = await runResolver(cssResolver, `./${id}`, base)
+    if (!dotResolved) throw new Error()
+    return dotResolved
+  } catch {
```

Comment:
Can we skip this if the path is missing an extension? ie. `@import "tailwindcss"`